app/src/main/java/org/akvo/flow/activity/FormActivity.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2010-2017 Stichting Akvo (Akvo Foundation)
+ * Copyright (C) 2010-2017,2018 Stichting Akvo (Akvo Foundation)
```

Comment:
this is technically correct. Note that since it was last edited in 2017 the shorthand version 2010-2018 expresses the same thing